src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests.vb

```diff
@@ -702,6 +702,47 @@ ICatchClause (Exception type: System.Exception) (OperationKind.CatchClause) (Syn
             VerifyOperationTreeAndDiagnosticsForTest(Of CatchBlockSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub TestGetOperationForEqualsValueVariableInitializer()
+            Dim source = <![CDATA[
+Class Test
+    Sub M()
+        Dim x = 1'BIND:"= 1"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableInitializer (OperationKind.VariableInitializer) (Syntax: '= 1')
+  ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of EqualsValueSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub TestGetOperationForAsNewVariableInitializer()
+            Dim source = <![CDATA[
+Class Test
+    Sub M()
+        Dim x As New Test'BIND:"As New Test"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableInitializer (OperationKind.VariableInitializer) (Syntax: 'As New Test')
```

Comment:
What syntax node do we use for a field initializer in a similar situation? Do we have tests for a scenario like that?

Reply:
We have a skipped test for multiple field initializer: [http://source.roslyn.io/#Roslyn.Compilers.VisualBasic.Semantic.UnitTests/IOperation/IOperationTests_ISymbolInitializer.vb,151](http://source.roslyn.io/#Roslyn.Compilers.VisualBasic.Semantic.UnitTests/IOperation/IOperationTests_ISymbolInitializer.vb,151). Let me add another test for single field initializer.

---
In reply to: [143259372](https://github.com/dotnet/roslyn/pull/22546#discussion_r143259372) [](ancestors = 143259372)

Reply:
Added a new unit test test, and the syntax seems to be as expected.

---
In reply to: [143265026](https://github.com/dotnet/roslyn/pull/22546#discussion_r143265026) [](ancestors = 143265026,143259372)